app/config/index.js

```diff
@@ -62,6 +89,19 @@ const config = {
     templateIdFarmerLogin: process.env.NOTIFY_TEMPLATE_ID_FARMER_LOGIN
   },
   port: process.env.PORT,
+  serviceName: process.env.SERVICE_NAME,
```

Comment:
This might be a merge issue as I have previously removed the setting of the service name in this way as I don't foresee it being needed.

Reply:
Removed